src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -166,6 +164,28 @@ struct fd_replay_tile_ctx {
   ulong       stake_weights_out_wmark;
   ulong       stake_weights_out_chunk;
 
+  // Inputs to plugin/gui
+
+  fd_frag_meta_t * replay_plugin_out_mcache;
```

Comment:
You don't need two different links here (replay_plugin, start_plugin), you can just use one.

Reply:
The replay link has a much larger message size than start. Does that matter?